cm/cm_hdd_cdd/input_layers.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
@Kanma I see that you add the documentation in the README file about the input_layers.json.
In my case the CM do not need any file to be selected from the user.
I think that use: 
```json
[
    {
        "dataset": "all",
    }
]
```
It is wrong because it require the user to select a layer that then is not used by the CM, therefore I think there should be a way to declare that the CM do not need any layer and user have to select only the region of interest.
What do you think? Should I open an issue?

Reply:
Hello, I'll make the necessary changes to support that.

Reply:
@zarch, the following changes are now in the `master`branch:

(from the README)

```
`dataset` is the ID of the dataset. The following special values are available:

* "all" can be used to specify all the datasets
* "all_rasters" can be used to specify all the raster datasets
* "all_vectors" can be used to specify all the vector datasets
* "none" can be used to indicates that the CM doesn't require any dataset at all
```